src/main/java/org/mifos/ops/zeebe/camel/routes/OperationsRouteBuilder.java

```diff
@@ -285,4 +340,8 @@ public void configure() {
                 .setBody(constant(null));
 
     }
+
+    private QueryBuilder getMatchPhraseQueryBuilder(String key, Object valueToMatch) {
```

Comment:
Is this required to be a separate function for a single line?

Reply:
it was used at multiple places so I separated it in function, incase any common thing is to be changed at all the places.

Reply:
If it's only going to be a single line of code in the function, then please do.